d1: update home page (#8959)
cloudflare/cloudflare-docs: content/d1/_index.md

{{</description>}}

> D1 is Cloudflare’s native serverless database. Create your first database by [following the getting started guide](/d1/get-started/), learn how to [importing data into a database](/d1/learning/importing-data/) or learn how to [query your database](/d1/platform/client-api/) directly from [Workers](/workers/) or [Pages](/pages/platform/functions/bindings/#d1-databases).

Reviewer: ```suggestion
D1 is Cloudflare’s native serverless database. Create your first D1 database by [following the Get started guide](/d1/get-started/), learn how to [import data into a database](/d1/learning/importing-data/), and how to [query your database](/d1/platform/client-api/) directly from [Workers](/workers/) or [Pages](/pages/platform/functions/bindings/#d1-databases).
```

{{<Aside type="warning" header="Open Alpha">}}

While in the [Open Alpha](https://blog.cloudflare.com/d1-open-alpha/) period, there is a possibility of breaking changes. The Alpha is meant for testing purposes and using it for production traffic is not recommended.

To report bugs or request features, go to the [Cloudflare Community Forums](https://community.cloudflare.com/c/developers/d1/85). To give feedback, go to the [D1 Discord channel](https://discord.com/invite/cloudflaredev). If you are having issues with Wrangler, report issues in the [Wrangler GitHub repository](https://github.com/cloudflare/workers-sdk/issues/new/choose).

{{</Aside>}}

## Get started
 
{{<feature header="Create your first database" href="/d1/get-started/">}}

> Learn how to create your first database, establish a schema, import data and query D1 directly from an application [built with Workers](/workers/).

Reviewer: ```suggestion
Learn how to create your first D1 database, establish a schema, import data and query D1 directly from an application [built with Workers](/workers/).
```

{{<plan type="all">}}

D1 is Cloudflare’s native serverless database. Create your first database by [following the getting started guide](/d1/get-started/), learn how to [importing data into a database](/d1/learning/importing-data/) or learn how to [query your database](/d1/platform/client-api/) directly from [Workers](/workers/) or [Pages](/pages/platform/functions/bindings/#d1-databases).

{{<Aside type="warning" header="Open Alpha">}}

While in the [Open Alpha](https://blog.cloudflare.com/d1-open-alpha/) period, there is a possibility of breaking changes. The Alpha is meant for testing purposes and using it for production traffic is not recommended.

To report bugs or request features, go to the [Cloudflare Community Forums](https://community.cloudflare.com/c/developers/d1/85). To give feedback, go to the [D1 Discord channel](https://discord.com/invite/cloudflaredev). If you are having issues with Wrangler, report issues in the [Wrangler GitHub repository](https://github.com/cloudflare/workers-sdk/issues/new/choose).

{{</Aside>}}

> ## Explore

Reviewer: ```suggestion
```
Agree with taking out Wrangler as a 'feature' as it isn't one but "Explore" is not a category in our Landing page guidelines. I'd recommend keeping the button as is without a section or make the section "# Get started" to avoid ambiguity.

"# Related products" should sit above the Workers and Pages links as per syle guide.

{{<feature header="Pages" href="/pages/">}}

> Deploy dynamic front-end applications and host static sites using Pages.

Reviewer: ```suggestion
Deploy dynamic front-end applications in record time.
```

I don't wanna be a downer but I'd like to mimic the Pages docs as much as possible to avoid multiple, different product descriptions. https://developers.cloudflare.com/pages/ I only removed "record time" because "record time" is a fuzzy metric.
